spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/ArrowVectorAccessorFactory.java

```diff
@@ -74,6 +76,11 @@ public UTF8String ofRow(VarCharVector vector, int rowId) {
           null, vector.getDataBuffer().memoryAddress() + start, end - start);
     }
 
+    @Override
+    public UTF8String ofRow(FixedSizeBinaryVector vector, int rowId) {
+      return UTF8String.fromString(UUIDUtil.convert(vector.get(rowId)).toString());
```

Comment:
Is there a way to get the underlying array and offset?

Reply:
`vector.get(rowId)` will return the byte[] with a length of 16 for the given `rowId`. I think we could get underlying array and offset from the underlying `ArrowBuf`, but we would need to read it into a new byte[], which is what `vector.get(rowId)` is doing underneath